src/reanimated2/screenTransition/swipeSimulator.ts

```diff
@@ -0,0 +1,228 @@
+'use strict';
+import type {
+  PanGestureHandlerEventPayload,
+  ScreenTransitionConfig,
+  RNScreensTurboModuleType,
+} from './commonTypes';
+import { applyStyle } from './styleUpdater';
+
+const RNScreensTurboModule: RNScreensTurboModuleType =
+  global.RNScreensTurboModule || {
+    startTransition: (_stackTag) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+      return {
+        topScreenId: -1,
+        belowTopScreenId: -1,
+        canStartTransition: false,
+      };
+    },
+    updateTransition: (_stackTag, _progress) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+    },
+    finishTransition: (_stackTag, _isCanceled) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
+    },
+  };
+
+function computeEasingProgress(
+  startingTimestamp: number,
+  distance: number,
+  velocity: number
+) {
+  'worklet';
+  if (Math.abs(distance) < 1) {
+    return 1;
+  }
+  const elapsedTime = (_getAnimationTimestamp() - startingTimestamp) / 1000;
```

Comment:
Not sure if we should call `_getAnimationTimestamp()` directly, maybe let's us  `_frameTimestamp` or something like that?

Reply:
Unfortunately the `_frameTimestamp` is undefined here :/